src/subgraph.ts

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios';
+let SUBGRAPH_URL = '';
+
+async function getCurrentExchangeRate() {
+    const query = `
+    {
+        stakePoolExchangeRates{
+          stkBnbToBnb
+          lastUpdatedBlock
+        }
+        }
+`;
+    let data;
```

Comment:
this isn't required to be declared, can be avoided

Reply:
done